ui/App.svelte

```diff
@@ -101,7 +100,7 @@
 <NotificationFaucet />
 <Theme />
 
-<Remote {store} context="session">
+<Remote {store} context="session" disableErrorLogging={true}>
```

Comment:
Why are we disabling the error logging here?

Reply:
If fetching the session errors, we show the blue screen. This means there is no need to show an additional notification.